src/en/charms-fan-aws-vpc.md

```diff
@@ -0,0 +1,69 @@
+Title: Creating an AWS VPC
+
+# Creating an AWS VPC
+
+This is in connection with [Juju and FAN networking][charms-fan] where an AWS
+user may wish to create an additional VPC (virtual private network).
+
+Note that only older AWS accounts will actually require a new VPC in order for
+Juju to autoconfigure the FAN. Recent AWS accounts are furnished with a VPC
+which will work out-of-the-box.
+
+The procedure covered here refers to the AWS web interface. Presumably it is
```

Comment:
I would omit the part about doing it with the API. Just saying presumably is a little un-authoratative.

Reply:
I'll reword and include a [link](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVpc.html).